rubocop/rubocop.yml

```diff
@@ -41,3 +41,6 @@ Metrics/BlockLength:
 Metrics/ModuleLength:
   Exclude:
     - 'spec/**/*'
+
+Style/BlockComments:
+  Enabled: false
```

Comment:
Based on your description comment, I'm not sure I follow where you see block comments in a spec helper (by default) that you want to retain. From the [documentation](http://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/BlockComments) describing block comments:

> This cop looks for uses of block comments (=begin…=end).
```
# bad
=begin
Multiple lines
of comments...
=end

# good
# Multiple lines
# of comments...
```

Moreover, while consistency throughout an application is important, a primary concern of this style configuration is for managing code that *we* write, not necessarily those of our dependencies (however, I understand the confusion/tension that generated/default code that doesn't adhere to the standard can create).

I personally don't see a need to introduce block comments, and am fine with the default of warning against them.